integration-tests/src/test/java/oracle/kubernetes/operator/utils/LoadBalancer.java

```diff
@@ -294,23 +306,31 @@ private void upgradeVoyagerNamespace() throws Exception {
   }
 
   private void createVoyagerIngress() throws Exception {
-    String chartDir = BaseTest.getProjectRoot() + "/integration-tests/src/test/resources/charts";
+    String chartDir = BaseTest.getProjectRoot() + "/kubernetes/samples/charts";
 
     StringBuffer cmd = new StringBuffer("cd ");
-    cmd.append(chartDir).append(" && helm install ingress-per-domain ");
-    cmd.append(" --name ")
-        .append(lbMap.get("name"))
-        .append(" --namespace ")
-        .append(lbMap.get("namespace"))
-        .append(" --set type=VOYAGER")
-        .append(" --set ")
-        .append("wlsDomain.domainUID=")
-        .append(lbMap.get("domainUID"))
-        .append(" --set ")
-        .append("wlsDomain.clusterName=")
-        .append(lbMap.get("clusterName"))
-        .append(" --set ")
-        .append("voyager.webPort=")
+    cmd.append(chartDir).append(" && ");
+
+    if (BaseTest.HELM_VERSION.equals("V2")) {
+      cmd.append(" helm install ingress-per-domain ")
```

Comment:
same comment as above

Reply:
There is no common syntax for both Helm Release